ChangeLog.d/fix-jpake-user-peer.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Allow setting user and peer identity for EC J-PAKE operation
+     instead role in PAKE PSA Crypto API.
```

Comment:
Perhaps add: "as described in the specification" to make it clear that it was a compliance bug.

Reply:
I will wait for the CI result and then fix it with other potential problems.

Reply:
CI was green. Force pushed fixed change log.